packages/analytics-js/src/app/RudderAnalytics.ts

```diff
@@ -168,7 +186,6 @@ class RudderAnalytics implements IRudderAnalytics<IAnalytics> {
 
       setExposedGlobal(GLOBAL_PRELOAD_BUFFER, clone(preloadedEventsArray));
 
-      this.analyticsInstances[writeKey] = new Analytics();
```

Comment:
This is not needed as `getAnalyticsInstance` call before will create a new instance if not already present.